javafx.plugin

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
I can't find an official source on this, but I would think all of the gradle stuff would be in the root directory.  I really just did it like that because of [the instructions for using the JavaFX plugin](http://speling.shemnon.com/blog/2013/10/29/javafx-gradle-plugin-0-dot-4-0-released/).